core/templates/pages/story-editor-page/editor-tab/story-editor.directive.ts

```diff
@@ -223,6 +224,11 @@ angular.module('oppia').directive('storyEditor', [
             }
           };
 
+          $scope.isNotesLengthExceeded = function(editableNotes) {
+            return (
+              editableNotes.length > $scope.maximumCharacterLengthOfNotes);
```

Comment:
Why not do this directly in the HTML considering bother editableNodes and the constant is available there?